pyatlan/generator/templates/methods/asset/a_p_i_path.jinja2

```diff
@@ -0,0 +1,11 @@
+
+    @classmethod
+    # @validate_arguments()
+    def create(cls, *, path_raw_u_r_i: str, spec_qualified_name: str) -> {{ entity_def.name }}:
```

Comment:
Ok, last one 🤞 — for these variable names (since we're writing them rather than relying on some generated code to pick them up like in the filename itself), let's not do the spaced-out naming for `snake_case`: so `path_raw_uri` instead of `path_raw_u_r_i` 🙏

Reply:
I get this error while just changing the characters like this on the `a_p_i_path` unit test
`>       assert sut.api_path_raw_uri == API_PATH_RAW_URI
E       AttributeError: 'APIPath' object has no attribute 'api_path_raw_uri'
`
Not sure why! Since i already defined the attributes in the attribute template
```
return {{ entity_def.name }}.Attributes(
                api_path_raw_uri=path_raw_uri,
   ```

Reply:
Solved it! My bad `api_path_raw_u_r_i` was defined as attribute as part of the template.